letsencrypt/client.py

```diff
@@ -191,29 +220,32 @@ def save_certificate(self, certr, cert_path, chain_path):
 
         return os.path.abspath(act_cert_path), cert_chain_abspath
 
-    def deploy_certificate(self, domains, privkey, cert_path, chain_path=None):
+    def deploy_certificate(self, domains, lineage):
         """Install certificate
 
         :param list domains: list of domains to install the certificate
 
-        :param privkey: private key for certificate
-        :type privkey: :class:`letsencrypt.le_util.Key`
-
-        :param str cert_path: certificate file path
-        :param str chain_path: chain file path
-
+        :param lineage: RenewableCert object representing the certificate
+        :type lineage: :class:`letsencrypt.storage.RenewableCert`
         """
         if self.installer is None:
             logging.warning("No installer specified, client is unable to deploy"
                             "the certificate")
             raise errors.LetsEncryptClientError("No installer available")
 
-        chain_path = None if chain_path is None else os.path.abspath(chain_path)
+        # TODO: Is it possible not to have a chain at all? (The
```

Comment:
can we put it as `.. todo::` into docstring and reference one of the related acme-spec issues?